Plots/XY/NCL_xy_2.py

```diff
@@ -0,0 +1,64 @@
+"""
+NCL_xy_2.py
+===============
+This script illustrates the following concepts:
+   - Drawing an XY plot with multiple curves
+   - Changing the line color for multiple curves in an XY plot
+   - Changing the line thickness for multiple curves in an XY plot
+   - Drawing XY plot curves with both lines and markers
+   - Changing the default markers in an XY plot
+   - Making all curves in an XY plot solid.
+See https://www.ncl.ucar.edu/Applications/xy.shtml for further information.
+"""
+
+###############################################################################
+# Import packages
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tic
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset("../../data/netcdf_files/uv300.nc")
+u = ds.U[:]
+
+###############################################################################
+# Generate plot data from netCDF data
+# xx = u.values[0,:,82]
```

Comment:
How about

```
da.isel(time=0, lon=82).plot(x="lat", marker='', markerfacecolor='blue', markersize=12, color='skyblue', linewidth=1)
da.isel(time=0, lon=69).plot(x="lat", marker='', color='red', linewidth=1, linestyle='dashed', label="toto")
```

Reply:
Agree with getting getting rid of additional variable (xx and yy) constructions and pushed that change as a commit. However, `xArray.DataArray.isel` alone does not work in this case because NCL code slices coordinates for "values" not integer indices. Therefore,  `xArray.DataArray.sel` is also used for that purpose.